apptest.go

```diff
@@ -471,7 +471,15 @@ func (a *AppSetup) waitForDeployedApps(ctx context.Context, apps []App) error {
 func (a *AppSetup) waitForDeployedApp(ctx context.Context, testApp App) error {
 	var err error
 
-	a.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("ensuring '%s/%s' app CR is %#q", testApp.AppCRNamespace, testApp.Name, deployedStatus))
+	var appCRNamespace string
+
+	if testApp.AppCRNamespace != "" {
+		appCRNamespace = testApp.AppCRNamespace
+	} else {
+		appCRNamespace = defaultNamespace
```

Comment:
Would it make sense to assign `testApp.Namespace`? 
I know it's the same but most of the tests are giving the `Namespace` value from their integration tests.

Reply:
testApp.Namespace is the target namespace where the helm release is created. It's not the namespace for the app CR.

This logic is the same as when the app CR is defined. So it will work even if we want to install the helm release is kube-system. 

 https://github.com/giantswarm/apptest/blob/1e082b17e5be8c560f9b0842ea25a94fdf7217d9/apptest.go#L316-L322